src/util/os_utils.js

```diff
@@ -317,6 +317,7 @@ function get_disk_mount_points() {
             dbg.log0('drives:', drives, ' current location ', process.cwd());
             return _.filter(drives, drive => {
                 const { mount, drive_id } = drive;
+                if (IS_DOCKER && mount === '/noobaa_storage') return true;
```

Comment:
The only issue I see with this, is that for the regular case that we have both mounts `/` and `/noobaa_storage` it will allow two drives to be filtered by this code, so while this workaround worked in the case where `/` was missing, I am not so sure it works correctly when it is present.

So I would fix it like so:
```js
if (IS_DOCKER) {
    return _.filter(drives, ({ mount }) => mount === '/noobaa_storage');
}
// continue as usual ...
return _.filter(drives, drive => {
```